frontend/src/services/effects.ts

```diff
@@ -277,6 +279,12 @@ export function withEffects(store: Store) {
         store.set('serviceAreas')([])
         store.set('selectedFilterMethod')('All')
       }
+      if (selectedDataset !== null && selectedDataset.name !== 'Your Data') {
+        store.set('allowDrivingTime')(true)
+      } else if (PUBLIC) {
+        store.set('allowDrivingTime')(false)
```

Comment:
does using the store make more sense then importing the env into the relevant file? i dunno

Reply:
That's what I did first but having it in the store makes it easier to control in `effects.ts`. It is not simply true if not public, false if public, since we need to listen on the selectedDataset

Reply:
What is `allowDrivingTime` set to if neither of these conditions are true?

Also, don't be afraid to move this new logic into a new effect if you think that would be cleaner.